lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,17 +552,26 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.warning(`The cloud instance discovery network request's status code is: ${response.status}`);
+                this.logger.warning("There weren't any cloud instances returned.");
+                
+                const tenantDiscoveryEnpoint = response.body.tenant_discovery_endpoint;
+                const tenantDiscoveryEnpointLoggerHelper = tenantDiscoveryEnpoint.length ? `: ${tenantDiscoveryEnpoint.length}` : " an empty string";
```

Comment:
```suggestion
                const tenantDiscoveryEnpointLoggerHelper = tenantDiscoveryEnpoint || " an empty string";
```

Reply:
I'm assuming you meant to use the actual endpoint and not the length of the string.